etl/src/energuide/flask_app.py

```diff
@@ -0,0 +1,81 @@
+import os
+import typing
+from http import HTTPStatus
+import hashlib
+import flask
+import pymongo
+from energuide import transform
+from energuide import database
+
+
+App = flask.Flask(__name__)
+
+
+DEFAULT_ETL_SECRET_KEY = 'no key'
+
+App.config.update(dict(
+    SECRET_KEY=os.environ.get('ETL_SECRET_KEY', DEFAULT_ETL_SECRET_KEY),
+))
+
+
+DATABASE_COORDS = database.DatabaseCoordinates(
+    username=os.environ.get(database.EnvVariables.username.value, default=database.EnvDefaults.username.value),
+    password=os.environ.get(database.EnvVariables.password.value, default=database.EnvDefaults.password.value),
+    host=os.environ.get(database.EnvVariables.host.value, default=database.EnvDefaults.host.value),
+    port=int(os.environ.get(database.EnvVariables.port.value, default=database.EnvDefaults.port.value))
+)
+
+DATABASE_NAME = os.environ.get(database.EnvVariables.database.value, default=database.EnvDefaults.database.value)
+
+COLLECTION = os.environ.get(database.EnvVariables.collection.value, default=database.EnvDefaults.collection.value)
+
+
+@App.route('/', methods=['GET'])
+def frontend() -> str:
+    return ''
+
+
+@App.route('/test_alive', methods=['GET'])
+def test_alive() -> str:
+    return 'Alive!'
+
+
+@App.route('/robots933456.txt', methods=['GET'])
+def robots() -> None:
+    flask.abort(HTTPStatus.NOT_FOUND)
+
+
+@App.route('/run_tl', methods=['POST'])
+def run_tl() -> typing.Tuple[str, int]:
```

Comment:
This seems to double up code from the load function in cli.py. Possibly necessary, but just an observation?